site/src/components/__tests__/footer.js

```diff
@@ -1,7 +1,15 @@
 import React from "react"
 import renderer from "react-test-renderer"
+import { render, fireEvent } from "react-testing-library"
 
 import Footer from "../footer"
+import LargeButton from "../largebutton"
```

Comment:
this doesn't look like its being used in the test itself, is it needed?

Reply:
AH good catch, will remove.